openreview/conference/templates/areachairWebfield.js

```diff
@@ -705,6 +705,7 @@ var postRenderTable = function(rows) {
             ddate: deleted ? Date.now() : null
           };
           body = view.getCopiedValues(body, paperRankingInvitation.reply);
+          $('.tag-widget').find('button').attr('disabled', true);
```

Comment:
It's more efficient to avoid using `find` when possible, so I would change this to `$('.tag-widget button')...`

Reply:
done